explorer/client/src/pages/object-result/views/PkgView.tsx

```diff
@@ -14,7 +14,6 @@ function PkgView({ data }: { data: DataType }) {
     const viewedData = {
         ...data,
         objType: trimStdLibPrefix(data.objType),
-        name: data.name,
```

Comment:
Yes - see the code for specifying the caption of the image pop out.

Reply:
i do see it, but why remove the name of a code package because of the caption of an image popout?

code packages do not have images associated to them

Reply:
The name of the code package is not being removed - please look at `src/pages/object-result/views/ObjectView.tsx`. The name of the package is inputted into the component `ObjHeader`.